docs-devsite/ai.md

```diff
@@ -264,6 +277,36 @@ export declare function getImagenModel(ai: AI, modelParams: ImagenModelParams, r
 
 If the `apiKey` or `projectId` fields are missing in your Firebase config.
 
+### getLiveGenerativeModel(ai, modelParams) {:#getlivegenerativemodel_f2099ac}
+
+> This API is provided as a preview for developers and may change based on feedback that we receive. Do not use this API in a production environment.
+> 
+
+Returns a [LiveGenerativeModel](./ai.livegenerativemodel.md#livegenerativemodel_class) class for real-time, bidirectional communication.
+
+The Live API is only supported in modern browser windows and Node &gt;<!-- -->= 22.
```

Comment:
Do we need to emphasize this limitation in the narrative guides?

Reply:
I don't think we need to emphasize this, since it's not very restrictive. The vast majority of browser users are using 'modern browsers'. Node users would be a very small subset of all Live users, and Node 22 is only one version above our minimum supported version.

@hsubox76 What do you think?
Supported environments: https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API#api.websocket